legal-api/src/legal_api/resources/v2/configuration.py

```diff
@@ -35,3 +36,114 @@ def get_configurations():
             configuration.json for configuration in configurations
         ]
     }), HTTPStatus.OK
+
+
+@bp.route('', methods=['PUT'])
+@cross_origin(origin='*')
+@jwt.has_one_of_roles([UserRoles.staff])
+def update_configurations():
+    """Update the configurations."""
+    json_input = request.get_json()
+    if not json_input:
+        return {'message': 'Request body cannot be blank'}, HTTPStatus.BAD_REQUEST
+
+    configurations = json_input.get('configurations', [])
+    error_message = validate_configurations(configurations)
+    if error_message:
+        return {'message': error_message}, HTTPStatus.BAD_REQUEST
+
+    response = []
+    try:
+        for config_data in configurations:
+            name = config_data.get('name')
+            value = config_data.get('value')
+            config = Configuration.find_by_name(name)
```

Comment:
Please update `Configuration.find_by_name` to be case insensitive.

Reply:
Updated

Reply:
Can we make this change in the `find_by_name` function instead?

Reply:
I made it at find_by_name too.
<img width="928" alt="Screenshot 2024-05-15 at 10 13 52 AM" src="https://github.com/bcgov/lear/assets/54437031/7fbb8b44-6a17-4ac6-ab06-fbda0104ed5c">